syncer/batchsync.go

```diff
@@ -122,14 +122,16 @@ func syncJobsStatus(storer jobs.Storer, status string, queues []string, job_summ
 				}
```

Comment:
This PR is more easily reviewed with [whitespace changes turned off](https://github.com/AdRoll/batchiepatchie/pull/110/files?diff=unified&w=1).